lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -1873,7 +1873,17 @@ func (d *zfs) ListVolumes() ([]Volume, error) {
 			v := NewVolume(d, d.name, volType, contentType, volName, make(map[string]string), d.config)
 
 			if isBlock {
-				v.SetMountFilesystemProbe(true)
+				// Get correct content type from lxd:content_type property.
+				contentType, err := d.getDatasetProperty(zfsVolName, "lxd:content_type")
```

Comment:
Can we somehow pull that property as part of the list command, otherwise we are going to be running potentially a lot of commands if there are a lot of block volumes?